### DISCUSSION TITLE
Advice on choosing a microphone

### DISCUSSION THREAD

**Discussion by Doudidou23** [mcguirepr89/BirdNET-Pi#921] [Q&A]
Hello,
I have read carefully your suggestions and feedbacks.
I'm new to the world of audio and after a successful test with a Rode mic from my camera I want to invest a little for with this material, is this a problem in your opinion?
Thank you!
![Capture](https://github.com/mcguirepr89/BirdNET-Pi/assets/134203136/1f8df741-4a1c-40d7-80dc-927fc0df7dcd)

**Comment by lloydbayley** [comment]
Wow! That's quite a list! Can't see much wrong with it. It's all quite reasonable in my opinion.
If I had to find any faults, I might say it's TOO good! :)
If you're happy with it, then I don't see why it shouldn't work!

**Comment by nilspupils** [comment]
I use the Behringer and it works very well! Have fun with your kit!

**Comment by lloydbayley** [comment]
I've always had good results with Behringer too. I think you'll like it.

**Comment by matthew73210** [comment]
In all honesty, any cheap usb mic will surfice. My 20 euro one can pick up birds in the forest 100m away. Over birds at the feeder. I wouldn't buy something so expensive right at the start. You could even try the network camera + rtsp route.

**Comment by Doudidou23** [comment]
Hello,
Thank you for your reply, I'm keeping this setup under wraps, but for the moment I've gone for a Rode AI-1 with a mirco condenser bought for less than 20$ on Amazon.
I'm waiting for an EM272 from England but the one on Amazon is already doing a great job!

**Comment by smauel2004** [comment]
Nice choice
